packages/sources/etherscan/README.md

```diff
@@ -0,0 +1,38 @@
+# Chainlink External Adapter for EtherScan
+
+### Environment Variables
+
+| Required? |  Name   |                            Description                            | Options | Defaults to |
+| :-------: | :-----: | :---------------------------------------------------------------: | :-----: | :---------: |
+|    ✅     | API_KEY | An API key that can be obtained [here](https://etherscan.io/apis) |         |             |
+
+---
+
+### Input Parameters
+
+| Required? |   Name   |     Description     |            Options             | Defaults to |
+| :-------: | :------: | :-----------------: | :----------------------------: | :---------: |
+|           | endpoint | The endpoint to use | [gasprice](#gasprice-Endpoint) | `gasprice`  |
+
+---
+
+## Gas Price Endpoint
+
+### Input Params
+
+| Required? |  Name   |    Description    |                     Options                     |  Defaults to   |
+| :-------: | :-----: | :---------------: | :---------------------------------------------: | :------------: |
+|    🟡     | `speed` | The desired speed | `SafeGasPrice`,`ProposeGasPrice`,`FastGasPrice` | `FastGasPrice` |
```

Comment:
Can we add aliases of these so that we could also use the same speed params as on the other gasprice adapters (in addition to the ones above)?